apps/meteor/client/views/admin/apps/AppsPage/AppsPage.tsx

```diff
@@ -25,49 +17,13 @@ const AppsPage = ({ isMarketplace }: AppsPageProps): ReactElement => {
 
 	const context = useRouteParameter('context');
 
-	const [isLoggedInCloud, setIsLoggedInCloud] = useState();
-
-	useEffect(() => {
-		const initialize = async (): Promise<void> => {
-			setIsLoggedInCloud(await checkUserLoggedIn());
-		};
-		initialize();
-	}, [checkUserLoggedIn]);
-
-	const handleLoginButtonClick = (): void => {
-		cloudRoute.push();
-	};
-
-	const handleUploadButtonClick = (): void => {
-		context && router.push({ context, page: 'install' });
-	};
-
 	const handleMarketplaceTabClick = (): void => router.push({ context: 'all', page: 'list' });
 
 	const handleInstalledTabClick = (): void => router.push({ context: 'installed', page: 'list' });
 
 	return (
 		<Page background='tint'>
-			<Page.Header title={t('Apps')}>
-				<ButtonGroup>
-					{isMarketplace && !isLoggedInCloud && (
-						<Button disabled={isLoggedInCloud === undefined} onClick={handleLoginButtonClick}>
-							{isLoggedInCloud === undefined ? (
-								<Skeleton width='x80' />
-							) : (
-								<>
-									<Icon name='download' /> {t('Login')}
-								</>
-							)}
-						</Button>
-					)}
-					{Boolean(isDevelopmentMode) && (
```

Comment:
removing this button I believe is out of scope here.  This is still something that needs to remain

Reply:
ok